docs/_data/mesheryctlcommands/pattern/delete/commands/delete.yaml

```diff
@@ -0,0 +1,4 @@
+name: delete
+description: delete pattern file will remove deployments of the pattern file using specific flags
```

Comment:
@navendu-pottekkat this command doesn't remove the deployment, but simply removes the pattern file, correct?

@utkarsh-pro do we have an endpoint and command plan for deprovisioning infrastructure using the `pattern` command?

Reply:
We don't have another endpoint. We use http method to indicate delete. Basically `POST` on `/api/experimental/pattern/deploy` will provision and `DELETE` on `/api/experimental/pattern/deploy` will deprovision.

Reply:
@utkarsh-pro very good. To clarify, do we have a way for the user to delete only the pattern file and not the infra that it represents?

Reply:
Yes @leecalcote. The above endpoint is only for handling `deployment`. We have separate CRUD endpoints for persisting pattern files. 
`/api/experimental/pattern` `POST GET`
`/api/experimental/pattern/:id` `GET DELETE`

These are the endpoints as far as I remember

Reply:
I made it as `delete pattern file will simply remove the pattern file using specific flags`
Is that correct now?

Reply:
Needs your approval to go!